.azure-pipelines.yml

```diff
@@ -1,4 +1,4 @@
-trigger:
+﻿trigger:
```

Comment:
We usually don't use those triggers internally, as the builds use a lot of resources and we do have limited resources. We should probably add an example of scheduled build trigger.

Reply:
```
schedules:
- cron: "30 08 * * Mon-Fri"
  displayName: 'Nightly weekdays build - 3:30 EST'
  always: false
  branches:
    include:
    - master
```